### PR TITLE
Feature/cc 757 sync dotfile

### PR DESCRIPTION
**Pull request by wplu**
https://jira.zenoss.com/browse/CC-757

from 10+ minutes on AWS zenoss.resmgr to 1 minute 30 seconds for initial startup.  Subsequent startups would be faster.

### CODE REVIEW

**Comment by jafcrocker** [comment]
My testing on a bare-metal box: 
resmgr initial startup: 40 sec
subsequent startups: 30 sec

**Comment by wplu** [comment]
thanks for that datapoint and WOW!

File: node/utils_test.go [hunk 1]
```diff
@@ -92,3 +102,90 @@ Last stable version: 0.6.6
...
+	// compare rsync'ed path from image against DFS volume
+	compareCmd := [...]string{"diff", "-qr", v.hostPath, expectedPath}
+	glog.V(2).Infof("compare command: %s", compareCmd)
+	docker = exec.Command(compareCmd[0], compareCmd[1:]...)
+	if output, err := docker.CombinedOutput(); err != nil {
+		t.Fatalf("could not compare paths: %+v, %s", compareCmd, string(output))
+	}
+
+	// compare user:group perms
+	getUidGidCmd := [...]string{"docker", "run", "--rm", v.image, "getent", "passwd", "games"}
```
Comment (wplu) [inline node/utils_test.go]: could be replaced with docker run -it ubuntu:12.04 bash -c "id -u games; id -g games"

**Comment by wplu** [comment]
replacing 'sleep 5s' with 'sync' will be on the next release:
  https://jira.zenoss.com/browse/CC-766

**Comment by wplu** [comment]
discussions going on - don't pull

**Comment by wplu** [comment]
cannot reopen due to rebase

**Pull request closed without merging**